src/devtools/views/Profiler/SidebarSelectedFiberInfo.css

```diff
@@ -10,6 +10,13 @@
 .Content {
   padding: 0.5rem;
   user-select: none;
+}
+
+.ContentShort {
+  overflow-y: visible;
```

Comment:
Hm...I don't think "overflow: visible" feels like the right solution here. I don't think we ever want this content to render outside of the padding box. I _think_ `flex-shrink: 0` might be a more appropriate fix here for each of the content sections, and then `overflow-y: auto` on the parent container (although not including the header bit). Thoughts?

Reply:
@bvaughn - unfortunately my knowledge of post-2010 CSS is pretty spotty so I'm the wrong guy to ask about whether `flex-shrink` would work here. I could try it out, but I'd worry that I might miss some cases.

Playing devil's advocate: the "short content" is always on top of the sidebar, and the sidebar is already clipped by the bottom of the browser frame. Even if the short content extends lower than the bottom, it won't affect what users see.  So even if using flex might be more correct, I'm not sure it's worth the effort until you start putting other can't-occlude content at the bottom of the sidebar. What do you think?

Reply:
I think the "correct" behavior is for the whole bottom area (excluding the title bar) to scroll as one, rather than individual `.Content` panels scrolling (since these don't _look_ like separate, scrollable panels- no border, etc)

I went ahead and made this change prior to merging your diff 😄 If you'd like, you can re-install and give it a spin and see how it feels.

Reply:
Haha, OK I'll try it out. BTW, this "why did this render?" feature is a winner. Debugging my toughest issues is much faster now. Also, beyond debugging, it's helped me learn about how React works-- I feel like I've learned more about how React works in the last few days than the previous few months. Thanks so much for building this!

Reply:
Nice! 😁 Thanks!